demos/matrix_free/stokes.py.rst

```diff
@@ -0,0 +1,50 @@
+==================
+ Stokes Equations
+==================
+Again, they are what they are.  This is the driven cavity example.
+We have several options files showing how to do some field-splitting
+and such for this.::
+
+  from firedrake import *
+
+  N = 128
+
+  M = UnitSquareMesh(N, N)
+
+  V = VectorFunctionSpace(M, "CG", 2)
+  W = FunctionSpace(M, "CG", 1)
+  Z = V * W
+
+  u, p = TrialFunctions(Z)
+  v, q = TestFunctions(Z)
+
+  a = (inner(grad(u), grad(v)) -
+       p * div(v) +
+       div(u) * q + p*q)*dx
```

Comment:
Why is there a pressure mass term here @rckirby?  I was expecting: `inner(grad(u), grad(v))*dx + div(v)*p*dx + q*div(u)*dx`.